doc/sphinx/source/ops/drop_out.rst

```diff
@@ -0,0 +1,70 @@
+.. drop_out.rst:
+
+#######
+DropOut
+#######
+
+.. code-block:: cpp
+
+   DropOut  // DropOut operations
+
+Description
+===========
+
+.. TODO
+
+Inputs
+------
+
```

Comment:
Let's remove the content of these boxes and the mathematical definition so we're not defining.
Maybe we could have a default stub page so we don't accidentally pull in other op doc.

Reply:
Good idea.  I was thinking it might be easier to write LaTEX kind of math when you have an example there already, but might just be confusing